pyproject.toml

```diff
@@ -43,9 +43,9 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "<3.12,>=3.7"
-structlog = "^22.3.0"
+structlog = "^21.2.0"
 PyYAML = "^6.0.0"
-pydantic = "^1.10.4"
+pydantic = ">=1.9.0"
```

Comment:
I'm not a fan of setting a lower bound without an upper bound. I'd suggest we use `^1.9` or `>=1.9.0,<2` instead.